fendermint/app/src/cmd/genesis.rs

```diff
@@ -344,10 +352,116 @@ pub async fn seal_genesis(genesis_file: &PathBuf, args: &SealGenesisArgs) -> any
     builder.write_to(args.output_path.clone()).await
 }
 
+/// Fetches F3 certificate data from the parent Filecoin chain
+async fn fetch_f3_params_from_parent(
```

Comment:
Should use `rust-f3` instead (if any functionality is missing, it can be added there)